### PR TITLE
[8.0] Add base_multi_tag module

### PR DESCRIPTION
**Pull request by katyukha**

### CODE REVIEW

File: base_multi_tag/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "website": "",
+    "summary": "Tags",
+    'category': 'Hidden/Dependency',
+    "depends": [
+        "base",
+        "base_action_rule",
+    ],
+    "description": """
+        Provides generic tags logic and basic views.
+        Integration with other modules should be implemented as sparate modules
```
Comment (dreispt) [inline base_multi_tag/__openerp__.py:14]: This probably too abstract.
Could you please provide a README with a better description?

**Comment by gurneyalex** [comment]
same remarks as in the 7.0 PR.

**Comment by gurneyalex** [comment]
I'm closing the PR as inactive.

**Comment by katyukha** [comment]
I am closing this PR.
If someone interested in this functionality, there is 8.0 branch of original [base_tags](https://github.com/katyukha/base_tags) addon, that seems to be working, but we do not use Odoo 8.0, so no updates planned.

**Pull request closed without merging**